scripts/wards.sh

```diff
@@ -25,9 +36,8 @@ for key in $(echo -e "$list" | sed "s/,/ /g")
 do
     contractName=$(seth --to-ascii "$key" | sed 's/\x0/ /g')
```

Comment:
```suggestion
    contractName=$(seth --to-ascii "$key")
```
This solves my issue running the script (seems only happening on mac). And honestly no idea why we need this `sed` command here.

Reply:
the mac issue could be related to this approach not working with all `sed` implementations ([ref](https://superuser.com/a/287998)).
the `sed` was introduced to address this issue on ubuntu:
`./scripts/wards.sh: line 37: warning: command substitution: ignored null byte in input`
the [issue](https://stackoverflow.com/q/46163678/8855986) seems to lead to the `seth` [command](https://github.com/dapphub/dapptools/blob/master/src/seth/libexec/seth/seth---to-ascii).  
I've investigated other approaches an refactored the script in `4459fab` to also simplify it and reduce code size.